src/aggregator/aggregator/handler/writer/protobuf.go

```diff
@@ -133,6 +133,7 @@ func (w *protobufWriter) prepare(mp aggregated.ChunkedMetricWithStoragePolicy) (
 	w.m.ID = append(w.m.ID, mp.Suffix...)
 	w.m.Metric.TimeNanos = mp.TimeNanos
 	w.m.Metric.Value = mp.Value
+	w.m.Annotation = mp.ChunkedMetric.Annotation
```

Comment:
Hm I think this should reuse memory like w.m.ID does (it copies the metric so that it doesn't need to worry about ownership from the caller).
```go
w.m.Annotation = append(w.m.Annotation[:0], mp.ChunkedMetric.Annotation...)
```